src/index.js

```diff
@@ -56,6 +59,12 @@ export default function (options = {}, callback = function noop () {}) {
       protractorArgs = filterArgs(protractorArgs)
       protractorArgs.push('--specs', specFiles.join(','))
     }
+
+    // If an alternative protractor config is specified, pass it in at the end of protractorArgs
```

Comment:
Nit: I think the logic here is clear enough that we don't need a comment but that's also very subjective :)

Reply:
You're right, probably unneeded - I might remove this in my next commit 😁